cmd/git-init/main.go

```diff
@@ -29,6 +29,7 @@ var (
 	revision               = flag.String("revision", "", "The Git revision to make the repository HEAD")
 	path                   = flag.String("path", "", "Path of directory under which git repository will be copied")
 	terminationMessagePath = flag.String("terminationMessagePath", "/dev/termination-log", "Location of file containing termination message")
+	submodules             = flag.Bool("submodules", true, "Initialize and fetch git submodules")
```

Comment:
WDYT about surfacing this option all the way up into the Git resource, so users can turn it off if they'd prefer?

Reply:
@ImJasonH I was waiting for someone to comment this to do it :stuck_out_tongue: